naptan_update/update_table.py

```diff
@@ -38,7 +37,7 @@ def lambda_handler():
     # Connect to the DynamoDB table
     table = dynamo_db.Table(TABLE_NAME)
```

Comment:
I can''t comment on the code before as it was inserted not in this PR - but last time wasn't reviewed - let's change the loop on all files like we did in the `create` no need to iterate over all files if we want one, just try ad open the `stops.csv`